team.html

```diff
@@ -67,18 +68,35 @@
                  </div>
             </div>
             <div class="row">
-                <h2>Project Director</h2>
+                <h2>Directors</h2>
+                <h3>Managing Director</h3>
                 <!-- <p class="inner">The Executive team consists of the project and technical director who oversee and organize all of the group's activities.</p> -->
-                 <div class="content-middle">
+                 <div class="content">
                     <div class="profile">
                         <div class="col-md-3">
-                            <img src="images/team/Jatin_Patil.jpg"/>
+                            <img src="images/team/James Oswald.png"/>
                         </div>
                         <div class="col-md-9">
-                            <h4>Jatin Patil</h4>
+                            <h4>James Oswald</h4>
                             <!-- <h6>Project Director</h6> -->
                             <p>
-                                Jatin is currently a 3rd year Nanotechnology Engineering student at the University of Waterloo. He has been on UW_NRG since his first year. He is passionate about research; he has done academic research in Waterloo, Harvard, and MIT. He is passionate about promoting science and engineering through planning and volunteering at outreach events.
+                                James is currently a 2nd year Nanotechnology Engineering student at the University of Waterloo. He has worked with the team since coming to the school. He is passionate about promoting science and engineering through research and development.
```

Comment:
Consider: "since coming to the school" -> "since the beginning of his university career", and maybe adding something more? This is quite basic.